src/NBitcoin/Block.cs

```diff
@@ -23,7 +23,7 @@ public class BlockHeader : IBitcoinSerializable
         internal const int Size = 80;
 
         /// <summary>Current header version.</summary>
-        public const int CurrentVersion = 7;
+        public virtual int CurrentVersion => 3;
```

Comment:
Why did the version go down here? Or am I reading it wrong ;)

Reply:
It was wrong before, this is the correct bitcoin version

Reply:
how about version 4?

Reply:
This came from NBitcoin so I assume its the btc current version, if its ver 4 now then this needs to change.

Reply:
What we need to do is make sure we comply with this version calculation
https://github.com/bitcoin/bitcoin/blob/598db389c33e5e90783ef1223df2eeab095ed622/src/validation.cpp#L1661

Reply:
Yes, it does not seem that there would be any such version any more like 2 3 or 4